tic_tac_toe/game.py

```diff
@@ -63,22 +101,45 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    winning_combinations = (
+        ((0, 0), (0, 1), (0, 2)),
+        ((1, 0), (1, 1), (1, 2)),
+        ((2, 0), (2, 1), (2, 2)),
+        ((0, 0), (1, 0), (2, 0)),
+        ((0, 1), (1, 1), (2, 1)),
+        ((0, 2), (1, 2), (2, 2)),
+        ((0, 0), (1, 1), (2, 2)),
+        ((0, 2), (1, 1), (2, 0))
+    )
+
+    if any(combination for combination in winning_combinations if _is_winning_combination(board, combination, player)):
```

Comment:
Actually it's exacly the same thing, but in one line. I wanted to try those one line things. That being said, I agree with you, it's confusing. Next time, I'll do normal for/if/return.